### PR TITLE
ORI-626: SSR Conversions through RTCM

### PR DESCRIPTION
**Pull request by mfine**
Convert the SSR messages through RTCM. Messages handled here:

```
1501 MSG_SSR_ORBIT_CLOCK
1510 MSG_SSR_PHASE_BIASES
1505 MSG_SSR_CODE_BIASES
1520 MSG_SSR_GRIDDED_CORRECTION
1525 MSG_SSR_GRID_DEFINITION
1515 MSG_SSR_STEC_CORRECTION
```

### CODE REVIEW

**Comment by mfine** [comment]
This is screaming for a table-driven approach.

File: c/src/sbp_rtcm3.c [hunk 1]
```diff
@@ -921,6 +921,96 @@ void sbp2rtcm_sbp_osr_cb(const u16 sender_id,
...
+                                     const u8 msg[],
+                                     struct rtcm3_out_state *state) {
+  rtcm_msg_swift_proprietary ssr_msg;
+  ssr_msg.msg_type = SBP_MSG_SSR_ORBIT_CLOCK;
+  ssr_msg.sender_id = sender_id;
+  ssr_msg.len = len;
+  memcpy(ssr_msg.data, msg, len);
+
+  u8 frame[RTCM3_MAX_MSG_LEN];
+  u16 frame_size = encode_rtcm3_frame(&ssr_msg, 4062, frame);
```
Comment (jbangelo) [inline c/src/sbp_rtcm3.c]: Did I forget to make a const variable for the `4062` message ID? We should probably make one.

**Comment by jbangelo** [comment]
Also, looks like unit tests are needed for these new functions?

File: c/src/sbp_rtcm3.c (hunk 1 above)
Comment (mfine) [reply c/src/sbp_rtcm3.c]: Convention in the project has been to use the message id's directly, not use defines or const variables. Going to continue with that.

**Comment by mfine** [comment]
Verified with the dev networks.

**Pull request merged**